cypress/e2e/pages/testsForAllPages.js

```diff
@@ -6,11 +6,7 @@
 import topicTagsTest from '../../support/helpers/topicTagsTest';
 import checkA11y from '../../support/helpers/checkA11y';
 
-export const testsThatAlwaysRunForAllPages = ({
-  service,
-  variant,
-  pageType,
-}) => {
+export default ({ service, variant, pageType }) => {
   describe(`testsToAlwaysRunForAllPages to run for ${service} ${pageType}`, () => {
```

Comment:
Let's also update the describe to match the file name.

We don't need to include service or page type in the message as we already have the URL

```suggestion
  describe(`testForAllPages`, () => {
```